samples/nrf9160/modem_shell/src/gnss/gnss.c

```diff
@@ -737,6 +742,7 @@ static void handle_timeout_work_fn(struct k_work *item)
 }
 
 #if defined(CONFIG_NRF_CLOUD_PGPS)
+#if !defined(CONFIG_NRF_CLOUD_MQTT)
```

Comment:
I was wondering where is the PGPS support?

Reply:
It just magically works. 😄 The P-GPS implementation fetches predictions when `nrf_cloud_pgps_init()`, `nrf_cloud_pgps_preemptive_updates()` or `nrf_cloud_pgps_notify_prediction()` is called (the so called indirect method).

https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/networking/nrf_cloud_pgps.html#requesting-and-processing-p-gps-data